src/styles/_dropdown.scss

```diff
@@ -368,9 +370,27 @@
         display: block;
       }
 
-      &.algolia-docsearch-suggestion__secondary .algolia-docsearch-suggestion--subcategory-column {
+      &.algolia-docsearch-suggestion__secondary {
         display: block;
       }
+
+      @media all and (min-width: #{$responsive-breakpoint}) {
+        .algolia-docsearch-suggestion--subcategory-column{
+          display: block;
+
+        }
+      }
+      @media all and (max-width: #{$responsive-breakpoint}) {
+        .algolia-docsearch-suggestion--subcategory-column{
+          display: none;
```

Comment:
I am not able to ensure this won't break any website by triggering this automatically. I think we discussed it a bit but now that I see it I am wondering if this is safe to propagate as it, thoughts?

Reply:
Yeah same as Vincent! 
Why did you removed the `.algolia-docsearch-suggestion--subcategory-column` ? 
I think the best way of doing it is to keep the `.algolia-docsearch-suggestion--subcategory-column {` without any media query, and remove it on ` (max-width: #{$responsive-breakpoint})`

Reply:
Sorry I haven't seen this comment before.

You were right, it could introduce a lost of display regarding the payload.

@LukyVj I have changed it before the PR of today. You shouldn't have seen it today.
https://github.com/algolia/docsearch/pull/197/files

Thanks guys